src/commands/createNewProject/verifyIsProject.ts

```diff
@@ -95,7 +95,7 @@ export async function verifyAndPromptToCreateProject(context: IActionContext, wo
             await context.ui.showWarningMessage(message, { modal: true, stepName: 'notAProject' }, DialogResponses.yes);
         }
 
-        options.folderPath = typeof workspaceFolder === 'string' ? workspaceFolder : workspaceFolder.uri.fsPath;
+        options.folderPath ||= typeof workspaceFolder === 'string' ? workspaceFolder : workspaceFolder.uri.fsPath;
```

Comment:
Assuming we want to do a patch release, I like this change because it works and it's simple.

Long term, though, I think we should do a bit of refactoring to prevent similar issues from happening in the future. For example
- Clean up some of the naming around `workspacePath` vs. `folderPath` vs. `projectPath`. IMO the issue came about because `workspacePath` [is set to](https://github.com/microsoft/vscode-azurefunctions/blob/main/src/commands/createFunction/createFunction.ts#L45) `options.folderPath` which clearly doesn't always match the workspace
- Stop using `workspaceFolder` as much (it was recommended by VS Code's `findFiles` api, but we [had problems](https://github.com/microsoft/vscode-azurefunctions/pull/2938) with that api anyways)